interface/wx/listctrl.h

```diff
@@ -256,6 +256,12 @@ enum
     @event{EVT_LIST_CACHE_HINT(id, func)}
            Prepare cache for a virtual list control.
            Processes a @c wxEVT_LIST_CACHE_HINT event type.
+    @event{EVT_LIST_ITEM_CHECKED(id, func)}
+           The item has been checked.
+           Processes a @c wxEVT_LIST_ITEM_CHECKED event type.
```

Comment:
Please do add "This event is available since wxWidgets 3.1.0" here too. Unfortunately I don't think you can use `@since` macro inside a paragraph here (but I could be wrong).